src/main/java/com/slisenko/education/html/PageDownloader.java

```diff
@@ -0,0 +1,43 @@
+package com.slisenko.education.html;
+
+import java.io.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Class for downloading resources from the internet
+ */
+public class PageDownloader {
+
+    private final int SIZE_BUF = 100000;
+
+    /**
+     * Downloads a resource from the internet and saves it in a file
+     * @param link Link to the resource
+     * @param path File path
+     * @return Returns true if successful, otherwise returns false
+     */
+    public void download(String link, String path) throws DownloadException {
+
+        URL url;
+
+        try {
+            url = new URL(link);
+        } catch (MalformedURLException e) {
+            throw new DownloadException("Error in the resource link");
```

Comment:
pass exception "e" as a constructor argument of DownloadException. also I suggest adding link to the message as well
for example throw new DownloadException("Error in the resource link" + link, e)

Reply:
Ok. Fixed.